java/arcs/android/storage/database/AndroidSqliteDatabaseRegistry.kt

```diff
@@ -163,7 +163,7 @@ class AndroidSqliteDatabaseRegistry(
         private val MANIFEST_NAME = "arcs_database_manifest"
 
         private val SCHEMA = """
-            CREATE TABLE arcs_databases (
+            CREATE TABLE IF NOT EXISTS arcs_databases (
```

Comment:
Are we confident that our tests drop the database (if needed) at their start? Otherwise this could introduce issues where tests unintentionally carry over state from others.

We could fix in follow-on if needed to unblock a release.

Reply:
Actually I saw such a race condition couple years ago.
The race condition was also asked frequently.
i.e. https://stackoverflow.com/questions/26127147/sqliteopenhelper-oncreate-running-twice

Reply:
@galganif should we file a follow-up issue to make sure this is not at risk of introducing the situation I describe above? It's possible I'm off base e.g. this is not used in our tests but I want to be thorough and check given we're already wrestling with test flakiness.

Reply:
this particular table, I don't think it would cause problems in tests in any case. whether we clean up databases correctly in tests I am not sure, but I don't think we have seen problems related to this. If anything they would manifest with the "SQLiteException: table already exists" message which clearly points to the problem, not random timeouts etc.